### PR TITLE
Update SideNav.js

### PR DESCRIPTION
**Pull request by alcolantonio**
Added volunteer signup link. Does this work?

### CODE REVIEW

File: client/src/Components/SideNav.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ class SideNav extends Component {
     let podcastUrl = 'https://soundcloud.com/user-640277634/sets/parity-podcast-season-2/s-Hka7a'
     let spreadsheetUrl = 'https://docs.google.com/spreadsheets/d/1F46H8ZRGP8Jzj1zSW0PT8HerBZ_BlHI5T48A2vp34r0/edit?usp=sharing'
     let srcUrl = 'https://github.com/kevinhughes27/parity-server'
+    let VolunteerUrl = 'https://docs.google.com/spreadsheets/d/1lunhlXMe5_sefD6Dy9OCreQxMtEi2mzDrmHLhBD7Elo/edit?usp=sharing'
```
Comment (kevinhughes27) [inline client/src/Components/SideNav.js]: annoying nitpicky comment - usually we make Javascript variables start with a lowercase and then camelCase the other words. For example `volunteerUrl`

File: client/src/Components/SideNav.js [hunk 2]
```diff
@@ -21,6 +22,7 @@ class SideNav extends Component {
         <li><Link to='/compare_players'>Compare Players</Link></li>
         <li><Link to='/team_dashboard'>Team Dashboard</Link></li>
         <li><Link to='/trade_simulator'>Trade Simulator</Link></li>
+        <li><Link to='/VolunteerUrl'>Volunteer</Link></li>
```
Comment (kevinhughes27) [inline client/src/Components/SideNav.js]: This currently links to a webpage at /VolunteerUrl which doesn't exist. The line of code that you want is down below:

```
<li><a href={volunteerUrl} target='_blank'> Volunteer</a></li>
```

The squiggly braces substitute the variable into the HTML while the other is just a plain string 😺

Comment (alcolantonio) [reply client/src/Components/SideNav.js]: Noted.

**Comment by kevinhughes27** [comment]
the link code is right. Now you need to change the variable name to use a lowercase `v` as well

**Comment by alcolantonio** [comment]
Done.

**Comment by alcolantonio** [comment]
Hamburger?

**Comment by kevinhughes27** [comment]
This will be live in a few minutes. You'll need to refresh the page. Thanks @alcolantonio!!

**Pull request merged**